<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gprs</class>
 <widget class="QWidget" name="Gprs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gprs</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>120</height>
        </size>
       </property>
       <property name="title">
        <string>com set</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>com:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="cb_com">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <item>
             <property name="text">
              <string>ttyO0</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ttyO1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ttyO2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ttyO3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ttyO4</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>bound:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QComboBox" name="cb_bound">
            <item>
             <property name="text">
              <string>115200</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>9600</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4800</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>databits:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QComboBox" name="cb_data">
            <property name="currentIndex">
             <number>3</number>
            </property>
            <item>
             <property name="text">
              <string>5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>6</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>7</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>8</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>stopbits:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="cb_stop">
            <item>
             <property name="text">
              <string>1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1.5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>parity:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QComboBox" name="cb_partity">
            <item>
             <property name="text">
              <string>N</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>O</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>E</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="4" colspan="2">
           <widget class="QPushButton" name="pbt_set">
            <property name="text">
             <string>set</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>gprs set</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="2">
           <widget class="QPushButton" name="pbt_send">
            <property name="text">
             <string>msg-s</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="pbt_gprs">
            <property name="text">
             <string>gprs</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>information:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QPushButton" name="ping">
            <property name="text">
             <string>ping</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="Edit_phone">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>phone:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Message:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="pbt_call">
            <property name="text">
             <string>call</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="hostname">
            <property name="text">
             <string>www.forlinx.com</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>hostname:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="Line_msg">
            <property name="text">
             <string>message text</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QTextEdit" name="Edit_info"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
